2023-05-07 01:15:52,160 INFO    [user_impl.py(logout:96)] log out: Miloš Marković
2023-05-07 01:18:54,515 ERROR   [movie_impl.py(repertoire_search:120)] Association proxy Movie.actors_association refers to an attribute 'genre' that is not directly mapped on class <class 'main.entities.models.movies_actors.MoviesActors'>; therefore this operation cannot proceed since we don't know what type of object is referred towards
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\movie_impl.py", line 94, in repertoire_search
    Movie.genres.any(Genre.name.like(f'%{search_input}%')),
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\ext\associationproxy.py", line 1141, in any
    self._ambiguous()
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\ext\associationproxy.py", line 1111, in _ambiguous
    raise AttributeError(
AttributeError: Association proxy Movie.actors_association refers to an attribute 'genre' that is not directly mapped on class <class 'main.entities.models.movies_actors.MoviesActors'>; therefore this operation cannot proceed since we don't know what type of object is referred towards

2023-05-07 01:19:12,848 ERROR   [movie_impl.py(repertoire_search:120)] Association proxy Movie.actors_association refers to an attribute 'genre' that is not directly mapped on class <class 'main.entities.models.movies_actors.MoviesActors'>; therefore this operation cannot proceed since we don't know what type of object is referred towards
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\movie_impl.py", line 87, in repertoire_search
    query = query.filter(Movie.genres.any(Genre.name == genre_name))
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\ext\associationproxy.py", line 1141, in any
    self._ambiguous()
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\ext\associationproxy.py", line 1111, in _ambiguous
    raise AttributeError(
AttributeError: Association proxy Movie.actors_association refers to an attribute 'genre' that is not directly mapped on class <class 'main.entities.models.movies_actors.MoviesActors'>; therefore this operation cannot proceed since we don't know what type of object is referred towards

2023-05-07 01:19:52,387 INFO    [user_impl.py(login:69)] login: shadow1
2023-05-07 01:19:52,387 INFO    [base_facade.py(find:13)] value: shadow1, column: username
2023-05-07 01:19:55,978 INFO    [user_impl.py(login:69)] login: shadow
2023-05-07 01:19:55,978 INFO    [base_facade.py(find:13)] value: shadow, column: username
2023-05-07 01:21:38,769 INFO    [base_facade.py(find:13)] value: S, column: title
2023-05-07 01:21:39,013 INFO    [base_facade.py(find:13)] value: St, column: title
2023-05-07 01:21:39,135 INFO    [base_facade.py(find:13)] value: Sta, column: title
2023-05-07 01:21:39,272 INFO    [base_facade.py(find:13)] value: Star, column: title
2023-05-07 01:21:39,496 INFO    [base_facade.py(find:13)] value: Star , column: title
2023-05-07 01:21:39,761 INFO    [base_facade.py(find:13)] value: Star W, column: title
2023-05-07 01:21:40,208 INFO    [base_facade.py(find:13)] value: Star Wa, column: title
2023-05-07 01:21:40,336 INFO    [base_facade.py(find:13)] value: Star War, column: title
2023-05-07 01:21:40,748 INFO    [base_facade.py(find:13)] value: Star Ward, column: title
2023-05-07 01:49:32,443 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 01:52:01,348 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 01:52:32,817 INFO    [base_facade.py(find:13)] value: '1', column: id
2023-05-07 01:52:44,419 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 01:55:51,661 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 13:34:52,824 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 13:36:29,011 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 13:36:29,035 INFO    [template_api.py(movie:21)] <Response 1092 bytes [200 OK]>
2023-05-07 13:37:14,695 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 13:37:52,165 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 13:37:52,191 INFO    [template_api.py(movie:22)] kurcina
2023-05-07 13:38:55,777 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 13:38:55,793 INFO    [movie_facade.py(find:14)] a
2023-05-07 13:38:55,802 INFO    [template_api.py(movie:22)] kurcina
2023-05-07 13:39:43,668 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 13:39:43,684 INFO    [movie_facade.py(find:14)] {'id': 1, 'title': 'Dracula Untold', 'year': '2014', 'duration': '1h 32m', 'rating': '6.2', 'votes': '104000', 'poster': 'dracula_untold.png', 'trailer': None}
2023-05-07 13:39:43,684 INFO    [movie_facade.py(find:15)] a
2023-05-07 13:39:43,692 INFO    [template_api.py(movie:22)] kurcina
2023-05-07 13:41:25,520 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 13:41:25,536 INFO    [movie_facade.py(find:14)] {'id': 1, 'title': 'Dracula Untold', 'year': '2014', 'duration': '1h 32m', 'rating': '6.2', 'votes': '104000', 'poster': 'dracula_untold.png', 'trailer': None}
2023-05-07 13:41:25,536 INFO    [movie_facade.py(find:15)] a
2023-05-07 13:41:25,536 INFO    [movie_facade.py(find:17)] b
2023-05-07 13:41:25,538 INFO    [movie_facade.py(find:21)] c
2023-05-07 13:41:25,539 INFO    [movie_facade.py(find:21)] c
2023-05-07 13:41:25,540 INFO    [movie_facade.py(find:21)] c
2023-05-07 13:41:25,541 INFO    [movie_facade.py(find:21)] c
2023-05-07 13:41:25,544 INFO    [movie_facade.py(find:25)] d
2023-05-07 13:41:25,544 INFO    [movie_facade.py(find:25)] d
2023-05-07 13:41:25,545 INFO    [movie_facade.py(find:25)] d
2023-05-07 13:41:25,548 INFO    [template_api.py(movie:22)] kurcina
2023-05-07 13:41:59,861 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 13:41:59,882 INFO    [movie_facade.py(find:14)] {'id': 1, 'title': 'Dracula Untold', 'year': '2014', 'duration': '1h 32m', 'rating': '6.2', 'votes': '104000', 'poster': 'dracula_untold.png', 'trailer': None}
2023-05-07 13:41:59,883 INFO    [movie_facade.py(find:15)] a
2023-05-07 13:41:59,883 INFO    [movie_facade.py(find:17)] b
2023-05-07 13:41:59,885 INFO    [movie_facade.py(find:21)] c
2023-05-07 13:41:59,886 INFO    [movie_facade.py(find:21)] c
2023-05-07 13:41:59,887 INFO    [movie_facade.py(find:21)] c
2023-05-07 13:41:59,889 INFO    [movie_facade.py(find:21)] c
2023-05-07 13:41:59,892 INFO    [movie_facade.py(find:25)] d
2023-05-07 13:41:59,893 INFO    [movie_facade.py(find:25)] d
2023-05-07 13:41:59,894 INFO    [movie_facade.py(find:25)] d
2023-05-07 13:44:14,343 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 13:45:18,023 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 13:49:15,346 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 13:53:26,030 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 14:02:29,515 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 14:02:55,611 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 14:02:55,613 INFO    [template_api.py(movie:22)] kurcina
2023-05-07 14:02:55,613 INFO    [template_api.py(movie:23)] {'id': 1, 'title': 'Dracula Untold', 'year': '2014', 'duration': '1h 32m', 'rating': '6.2', 'votes': '104000', 'poster': 'dracula_untold.png', 'trailer': None, 'actors': [{'id': 1, 'name': 'Luke Evans', 'image': 'luke_evans.jpg', 'role': 'Vlad'}, {'id': 2, 'name': 'Sarah Gadon', 'image': 'sarah_gadon.png', 'role': 'Mirena'}, {'id': 3, 'name': 'Dominic Cooper', 'image': 'dominic_cooper.png', 'role': 'Mehmed'}, {'id': 4, 'name': 'Charles Dance', 'image': 'charles_dance.png', 'role': 'Master Vampire'}], 'genres': [{'id': 1, 'name': 'Action', 'image': 'action.png'}, {'id': 3, 'name': 'Drama', 'image': 'drama.png'}, {'id': 4, 'name': 'Fantasy', 'image': 'fantasy.png'}]}
2023-05-07 14:07:20,010 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 14:07:20,026 ERROR   [base_impl.py(__result_handler__:32)] Object of type Result is not JSON serializable
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 30, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\utility\utils.py", line 8, in json
    json_data = dumps(entity, sort_keys=False)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Result is not JSON serializable

2023-05-07 14:14:03,741 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 14:14:03,756 INFO    [template_api.py(movie:22)] kurcina
2023-05-07 14:14:03,757 INFO    [template_api.py(movie:23)] {'id': 1, 'title': 'Dracula Untold', 'year': '2014', 'duration': '1h 32m', 'rating': '6.2', 'votes': '104000', 'poster': 'dracula_untold.png', 'trailer': None, 'actors': [{'id': 1, 'name': 'Luke Evans', 'image': 'luke_evans.jpg', 'role': 'Vlad'}, {'id': 2, 'name': 'Sarah Gadon', 'image': 'sarah_gadon.png', 'role': 'Mirena'}, {'id': 3, 'name': 'Dominic Cooper', 'image': 'dominic_cooper.png', 'role': 'Mehmed'}, {'id': 4, 'name': 'Charles Dance', 'image': 'charles_dance.png', 'role': 'Master Vampire'}], 'genres': [{'id': 1, 'name': 'Action', 'image': 'action.png'}, {'id': 3, 'name': 'Drama', 'image': 'drama.png'}, {'id': 4, 'name': 'Fantasy', 'image': 'fantasy.png'}]}
2023-05-07 14:14:05,347 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 14:14:05,350 ERROR   [base_impl.py(__result_handler__:32)] Object of type Result is not JSON serializable
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 30, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\utility\utils.py", line 8, in json
    json_data = dumps(entity, sort_keys=False)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Result is not JSON serializable

2023-05-07 14:15:17,353 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 14:15:17,377 ERROR   [base_impl.py(__result_handler__:32)] Object of type Result is not JSON serializable
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 30, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\utility\utils.py", line 9, in json
    json_data = dumps(entity, sort_keys=False)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Result is not JSON serializable

2023-05-07 14:22:20,524 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 14:22:20,543 ERROR   [base_impl.py(__result_handler__:32)] Object of type Result is not JSON serializable
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 30, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\utility\utils.py", line 9, in json
    json_data = dumps(entity, sort_keys=False)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Result is not JSON serializable

2023-05-07 14:25:27,132 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 14:25:27,156 ERROR   [base_impl.py(__result_handler__:32)] Object of type Result is not JSON serializable
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 30, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\utility\utils.py", line 9, in json
    json_data = dumps(entity, sort_keys=False)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Result is not JSON serializable

2023-05-07 14:33:25,220 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 14:33:25,244 ERROR   [base_impl.py(__result_handler__:32)] Object of type Result is not JSON serializable
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 30, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\utility\utils.py", line 9, in json
    json_data = dumps(entity, sort_keys=False)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Result is not JSON serializable

2023-05-07 14:42:40,954 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 14:42:40,976 ERROR   [base_impl.py(__result_handler__:32)] 'Result' object has no attribute 'items'
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 30, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\utility\utils.py", line 8, in json
    replace_none_with_null(entity)
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\utility\utils.py", line 14, in replace_none_with_null
    for key, value in data.items():
AttributeError: 'Result' object has no attribute 'items'

2023-05-07 14:43:29,460 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 14:43:29,484 ERROR   [base_impl.py(__result_handler__:32)] Object of type Result is not JSON serializable
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 30, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\utility\utils.py", line 9, in json
    json_data = dumps(entity, sort_keys=False)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Result is not JSON serializable

2023-05-07 14:43:55,292 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 14:43:55,318 ERROR   [base_impl.py(__result_handler__:32)] Object of type Result is not JSON serializable
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 30, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\utility\utils.py", line 10, in json
    json_data = dumps(entity, sort_keys=False)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Result is not JSON serializable

2023-05-07 14:53:59,430 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 14:53:59,450 ERROR   [base_impl.py(__result_handler__:32)] maximum recursion depth exceeded
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 30, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\utility\utils.py", line 11, in json
    return Response(json(entity), content_type='application/json')
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\utility\utils.py", line 11, in json
    return Response(json(entity), content_type='application/json')
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\utility\utils.py", line 11, in json
    return Response(json(entity), content_type='application/json')
  [Previous line repeated 976 more times]
RecursionError: maximum recursion depth exceeded

2023-05-07 14:57:57,572 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 14:58:20,525 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:00:42,587 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:01:31,546 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:01:55,006 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:02:16,736 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:02:50,476 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:03:46,012 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:09:22,396 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:09:22,413 ERROR   [base_impl.py(__result_handler__:32)] Object of type Result is not JSON serializable
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 30, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\utility\utils.py", line 10, in json
    json_data = dumps(entity, sort_keys=False)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Result is not JSON serializable

2023-05-07 15:10:46,361 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:10:46,385 ERROR   [base_impl.py(__result_handler__:32)] Object of type Result is not JSON serializable
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 30, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\utility\utils.py", line 10, in json
    json_data = dumps(entity, sort_keys=False)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Result is not JSON serializable

2023-05-07 15:11:52,312 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:11:52,329 ERROR   [base_impl.py(__result_handler__:32)] Object of type Result is not JSON serializable
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 30, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\utility\utils.py", line 10, in json
    json_data = dumps(entity, sort_keys=False)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Result is not JSON serializable

2023-05-07 15:12:18,933 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:12:18,950 ERROR   [base_impl.py(__result_handler__:32)] Object of type Result is not JSON serializable
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 30, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\utility\utils.py", line 10, in json
    json_data = dumps(entity, sort_keys=False)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Result is not JSON serializable

2023-05-07 15:14:28,630 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:14:28,661 ERROR   [base_impl.py(__result_handler__:32)] Object of type Result is not JSON serializable
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 30, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\utility\utils.py", line 10, in json
    json_data = dumps(entity, sort_keys=False)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Result is not JSON serializable

2023-05-07 15:16:10,027 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:17:16,550 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:27:52,902 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:27:54,264 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:28:15,976 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:28:15,978 INFO    [template_api.py(movie:22)] kurcina
2023-05-07 15:28:15,979 INFO    [template_api.py(movie:23)] {'id': 1, 'title': 'Dracula Untold', 'year': '2014', 'duration': '1h 32m', 'rating': '6.2', 'votes': '104000', 'poster': 'dracula_untold.png', 'trailer': None, 'actors': [{'id': 1, 'name': 'Luke Evans', 'image': 'luke_evans.jpg', 'role': 'Vlad'}, {'id': 2, 'name': 'Sarah Gadon', 'image': 'sarah_gadon.png', 'role': 'Mirena'}, {'id': 3, 'name': 'Dominic Cooper', 'image': 'dominic_cooper.png', 'role': 'Mehmed'}, {'id': 4, 'name': 'Charles Dance', 'image': 'charles_dance.png', 'role': 'Master Vampire'}], 'genres': [{'id': 1, 'name': 'Action', 'image': 'action.png'}, {'id': 3, 'name': 'Drama', 'image': 'drama.png'}, {'id': 4, 'name': 'Fantasy', 'image': 'fantasy.png'}]}
2023-05-07 15:34:07,584 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:34:07,599 INFO    [template_api.py(movie:24)] {'id': 1, 'title': 'Dracula Untold', 'year': '2014', 'duration': '1h 32m', 'rating': '6.2', 'votes': '104000', 'poster': 'dracula_untold.png', 'trailer': None, 'actors': [{'id': 1, 'name': 'Luke Evans', 'image': 'luke_evans.jpg', 'role': 'Vlad'}, {'id': 2, 'name': 'Sarah Gadon', 'image': 'sarah_gadon.png', 'role': 'Mirena'}, {'id': 3, 'name': 'Dominic Cooper', 'image': 'dominic_cooper.png', 'role': 'Mehmed'}, {'id': 4, 'name': 'Charles Dance', 'image': 'charles_dance.png', 'role': 'Master Vampire'}], 'genres': [{'id': 1, 'name': 'Action', 'image': 'action.png'}, {'id': 3, 'name': 'Drama', 'image': 'drama.png'}, {'id': 4, 'name': 'Fantasy', 'image': 'fantasy.png'}]}
2023-05-07 15:35:49,556 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:35:49,574 INFO    [template_api.py(movie:24)] {'id': 1, 'title': 'Dracula Untold', 'year': '2014', 'duration': '1h 32m', 'rating': '6.2', 'votes': '104000', 'poster': 'dracula_untold.png', 'trailer': None, 'actors': [{'id': 1, 'name': 'Luke Evans', 'image': 'luke_evans.jpg', 'role': 'Vlad'}, {'id': 2, 'name': 'Sarah Gadon', 'image': 'sarah_gadon.png', 'role': 'Mirena'}, {'id': 3, 'name': 'Dominic Cooper', 'image': 'dominic_cooper.png', 'role': 'Mehmed'}, {'id': 4, 'name': 'Charles Dance', 'image': 'charles_dance.png', 'role': 'Master Vampire'}], 'genres': [{'id': 1, 'name': 'Action', 'image': 'action.png'}, {'id': 3, 'name': 'Drama', 'image': 'drama.png'}, {'id': 4, 'name': 'Fantasy', 'image': 'fantasy.png'}]}
2023-05-07 15:36:03,231 INFO    [base_facade.py(find:13)] value: 7, column: id
2023-05-07 15:36:03,232 INFO    [template_api.py(movie:24)] False
2023-05-07 15:36:13,232 INFO    [base_facade.py(find:13)] value: 3, column: id
2023-05-07 15:36:13,233 INFO    [template_api.py(movie:24)] {'id': 3, 'title': 'Child 44', 'year': '2015', 'duration': '2h 17m', 'rating': '6.4', 'votes': '70000', 'poster': 'child_44.png', 'trailer': None, 'actors': [{'id': 9, 'name': 'Tom Hardy', 'image': 'tom_hardy.jpg', 'role': 'Leo Demidov'}, {'id': 11, 'name': 'Josef Altin', 'image': 'josef_altin.png', 'role': 'Alexander'}, {'id': 12, 'name': 'Fares Fares', 'image': 'fares_fares.jpg', 'role': 'Alexei Andreyev'}, {'id': 13, 'name': 'Jason Clarke', 'image': 'jason_clarke.jpg', 'role': 'Anatoly Brodsky'}, {'id': 15, 'name': 'Gary Oldman', 'image': 'gary_oldman.jpg', 'role': 'Mikhail Nesterov'}, {'id': 16, 'name': 'Noomi Rapace', 'image': 'noomi_rapace.jpg', 'role': 'Raisa Demidova'}, {'id': 17, 'name': 'Joel Kinnaman', 'image': 'joel_kinnaman.png', 'role': 'Vasili Nikitin'}, {'id': 18, 'name': 'Vincent Cassel', 'image': 'vincent_cassel.png', 'role': 'Major Kuzmin'}, {'id': 19, 'name': 'Paddy Considine', 'image': 'paddy_considine.jpg', 'role': 'Vladimir Malevich'}], 'genres': [{'id': 3, 'name': 'Drama', 'image': 'drama.png'}, {'id': 6, 'name': 'History', 'image': 'history.png'}, {'id': 7, 'name': 'Crime', 'image': 'crime.png'}]}
2023-05-07 15:38:05,976 INFO    [base_facade.py(find:13)] value: 3, column: id
2023-05-07 15:38:09,483 INFO    [base_facade.py(find:13)] value: 10, column: id
2023-05-07 15:38:13,062 INFO    [base_facade.py(find:13)] value: 5, column: id
2023-05-07 15:38:18,884 INFO    [base_facade.py(find:13)] value: 6, column: id
2023-05-07 15:38:21,808 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 15:39:59,124 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 15:40:02,603 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 15:40:06,058 INFO    [base_facade.py(find:13)] value: 6, column: id
2023-05-07 15:42:50,254 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 15:43:00,468 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 16:55:39,824 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 16:58:01,301 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 16:58:33,732 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 17:04:06,198 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 17:08:32,869 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 17:09:08,454 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 17:10:10,327 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 17:11:39,168 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 17:12:23,715 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 18:07:03,613 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:08:25,554 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:09:53,528 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:13:24,775 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:13:33,565 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:13:53,917 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:24:53,398 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:25:00,411 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:26:20,788 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:26:32,623 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:26:44,263 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:26:52,209 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:27:10,336 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:27:25,402 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:27:46,131 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:28:06,194 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:28:22,845 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:28:51,641 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:30:29,853 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 18:32:28,040 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 18:33:15,429 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 18:33:40,248 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 18:33:51,154 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 18:34:01,232 INFO    [base_facade.py(find:13)] value: 2, column: id
2023-05-07 18:34:11,264 INFO    [base_facade.py(find:13)] value: 3, column: id
2023-05-07 18:34:43,728 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:35:35,467 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:36:12,536 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:36:26,247 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:36:38,753 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:36:43,890 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:36:46,740 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:37:00,663 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:39:50,288 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:41:22,600 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:42:03,717 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:42:18,484 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:44:24,827 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:44:54,096 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:46:07,251 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:46:27,416 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:46:42,685 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:47:27,614 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:48:24,971 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:48:36,810 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:49:33,044 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:49:55,620 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:50:03,738 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:50:32,170 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:50:56,114 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:52:36,665 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:53:57,817 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:54:20,517 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:55:14,418 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:55:51,986 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:55:57,582 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:56:11,135 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:56:13,266 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:56:31,545 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:56:46,953 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:56:53,401 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 18:57:11,457 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:01:18,101 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:01:39,925 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:01:56,124 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:02:27,725 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:03:09,122 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:04:05,423 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:04:13,670 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:04:55,860 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:06:30,787 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:07:02,161 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:07:17,229 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:08:02,374 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:08:13,308 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:08:38,842 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:09:08,335 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:09:28,233 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:10:02,452 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:10:34,384 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:10:42,984 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:11:05,439 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:14:20,138 ERROR   [base_impl.py(__result_handler__:31)] __repr__ returned non-string (type dict)
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 29, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\entities\core\result.py", line 46, in response
    json_data = dumps(self.__repr__(), sort_keys=False, indent=4)
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\entities\core\result.py", line 42, in __repr__
    "item": self._item.__repr__()
TypeError: __repr__ returned non-string (type dict)

2023-05-07 19:14:27,533 ERROR   [base_impl.py(__result_handler__:31)] __repr__ returned non-string (type dict)
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 29, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\entities\core\result.py", line 46, in response
    json_data = dumps(self.__repr__(), sort_keys=False, indent=4)
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\entities\core\result.py", line 42, in __repr__
    "item": self._item.__repr__()
TypeError: __repr__ returned non-string (type dict)

2023-05-07 19:14:30,340 ERROR   [base_impl.py(__result_handler__:31)] __repr__ returned non-string (type dict)
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 29, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\entities\core\result.py", line 46, in response
    json_data = dumps(self.__repr__(), sort_keys=False, indent=4)
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\entities\core\result.py", line 42, in __repr__
    "item": self._item.__repr__()
TypeError: __repr__ returned non-string (type dict)

2023-05-07 19:23:56,553 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:24:03,051 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:24:14,295 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:24:17,955 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:24:18,137 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:25:36,216 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:25:36,239 ERROR   [base_impl.py(__result_handler__:31)] JSONEncoder.__init__() got an unexpected keyword argument 'encoding'
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 29, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\entities\core\result.py", line 46, in response
    json_data = dumps(self.__repr__(), sort_keys=False, indent=4, ensure_ascii=False, encoding='utf-8')
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 234, in dumps
    return cls(
TypeError: JSONEncoder.__init__() got an unexpected keyword argument 'encoding'

2023-05-07 19:25:40,689 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:25:40,691 ERROR   [base_impl.py(__result_handler__:31)] JSONEncoder.__init__() got an unexpected keyword argument 'encoding'
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 29, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\entities\core\result.py", line 46, in response
    json_data = dumps(self.__repr__(), sort_keys=False, indent=4, ensure_ascii=False, encoding='utf-8')
  File "C:\Users\Markovic Milos\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 234, in dumps
    return cls(
TypeError: JSONEncoder.__init__() got an unexpected keyword argument 'encoding'

2023-05-07 19:25:56,616 INFO    [base_facade.py(find:13)] value: 1, column: id
2023-05-07 19:29:10,807 ERROR   [base_impl.py(__result_handler__:31)] __repr__ returned non-string (type dict)
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\service\impl\base_impl.py", line 29, in __result_handler__
    return result.response()
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\entities\core\result.py", line 46, in response
    json_data = dumps(self.__repr__(), sort_keys=False, indent=4, ensure_ascii=False)
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\entities\core\result.py", line 42, in __repr__
    "item": self._item.__repr__()
TypeError: __repr__ returned non-string (type dict)

2023-05-07 19:30:11,064 ERROR   [base_facade.py(find_all:46)] __repr__ returned non-string (type dict)
Traceback (most recent call last):
  File "C:\Users\Markovic Milos\PycharmProjects\pythonProject\zavrsni-rad\main\entities\facade\base_facade.py", line 43, in find_all
    log.info(str(response))
TypeError: __repr__ returned non-string (type dict)

